pkg/util/encryption/aead.go

```diff
@@ -6,4 +6,5 @@ package encryption
 type AEAD interface {
 	Open([]byte) ([]byte, error)
 	Seal([]byte) ([]byte, error)
+	GetSealSecretVersion() string
```

Comment:
In go there is convention to not have prefixes for [getters](https://golang.org/doc/effective_go#Getters).